tests/test_transforms/sklearn/test_interface.py

```diff
@@ -0,0 +1,270 @@
+        RobustScalerTransform,
+        MinMaxScalerTransform,
+    ],
+)
+@pytest.mark.parametrize(
+    "in_column",
+    [
+        "exog_1",
+        ["exog_1", "exog_2"],
+    ],
+)
+def test_generated_column_names(transform_constructor, in_column, multicolumn_ts):
+    """Test that transform generates names for the columns correctly."""
+    transform = transform_constructor(in_column=in_column, out_column=None, inplace=False)
+    initial_df = multicolumn_ts.to_pandas()
+    transformed_df = transform.fit_transform(multicolumn_ts.to_pandas())
+    segments = sorted(multicolumn_ts.segments)
+
+    new_columns = extract_new_features_columns(transformed_df, initial_df)
+
+    for column in new_columns:
+        # create transform from column
+        transform_temp = eval(column)
+        df_temp = transform_temp.fit_transform(multicolumn_ts.to_pandas())
+        columns_temp = (
+            df_temp.columns.get_level_values("feature")
+            .difference(initial_df.columns.get_level_values("feature"))
+            .unique()
+            .tolist()
+        )
+
+        # compare column names and column values
+        assert len(columns_temp) == 1
+        column_temp = columns_temp[0]
+        assert column_temp == column
+        assert np.all(
+            df_temp.loc[:, pd.IndexSlice[segments, column_temp]]
+            == transformed_df.loc[:, pd.IndexSlice[segments, column]]
+        )
+
+
+@pytest.mark.parametrize(
+    "transform_constructor",
+    [
+        BoxCoxTransform,
+        YeoJohnsonTransform,
+        StandardScalerTransform,
+        RobustScalerTransform,
+        MinMaxScalerTransform,
+        MaxAbsScalerTransform,
+        StandardScalerTransform,
+        RobustScalerTransform,
+        MinMaxScalerTransform,
+    ],
+)
+def test_all_columns(transform_constructor, multicolumn_ts):
+    """Test that transform can process all columns using None value for in_column."""
+    transform = transform_constructor(in_column=None, out_column=None, inplace=False)
+    initial_df = multicolumn_ts.df.copy()
+    transformed_df = transform.fit_transform(multicolumn_ts.df)
+
+    new_columns = extract_new_features_columns(transformed_df, initial_df)
+    assert len(new_columns) == initial_df.columns.get_level_values("feature").nunique()
+
+
+@pytest.mark.parametrize(
+    "transform_constructor",
+    [
+        BoxCoxTransform,
+        YeoJohnsonTransform,
+        StandardScalerTransform,
+        RobustScalerTransform,
+        MinMaxScalerTransform,
+        MaxAbsScalerTransform,
+        StandardScalerTransform,
+        RobustScalerTransform,
+        MinMaxScalerTransform,
+    ],
+)
+@pytest.mark.parametrize(
+    "in_column", [["exog_1", "exog_2", "exog_3"], ["exog_2", "exog_1", "exog_3"], ["exog_3", "exog_2", "exog_1"]]
+)
+@pytest.mark.parametrize(
+    "mode",
+    [
+        "macro",
+        "per-segment",
+    ],
+)
+def test_ordering(transform_constructor, in_column, mode, multicolumn_ts):
```

Comment:
Could you also write a test that checks that out_columns[i] is output column for in_column[i] or get the output column in line 263 by the index in out_columns